vsintegration/src/FSharp.Editor/Commands/HelpContextService.fs

```diff
@@ -26,7 +26,7 @@ type internal FSharpHelpContextService
 
     static member GetHelpTerm(checker: FSharpChecker, sourceText : SourceText, fileName, options, span: TextSpan, tokens: List<ClassifiedSpan>, textVersion) : Async<string option> = 
         asyncMaybe {
-            let! _, check = checker.ParseAndCheckDocument(fileName, textVersion, sourceText.ToString(), options)
+            let! _, _, check = checker.ParseAndCheckDocument(fileName, textVersion, sourceText.ToString(), options)
```

Comment:
It's best to be explicit about ``allowStaleResults`` at each callsite, making the argument non-optional and always named at each callsite